corehq/apps/app_manager/util.py

```diff
@@ -328,6 +329,17 @@ def get_case_properties(app, case_types, defaults=(),
     )
 
 
+def get_shared_case_types(app):
+    shared_case_types = set()
+
+    if app.case_sharing:
+        apps = get_case_sharing_apps_in_domain(app.domain, app.id)
```

Comment:
nit: this would be more self documenting if you passed the app ID as a kwarg:
```
get_case_sharing_apps_in_domain(domain, exclude_app_id=app.id)
```
This way I don't have to go and look at the function definition to know what it's doing with the app ID.